### ISSUE TITLE
New transcribers go straight to training syndicate

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#2042]
When we recruit a new transcriber we say that they should work on a county in which they have an interest.  But when they register they are presented with a list of Syndicates who are looking for transcribers., including "Transcribe anywhere" and "Rest of England".
Also, if they go to Jo for training, she then has to know in which county they are interested and which have data.
So I suggest that when a transcriber registers, they also select a county in which they are interested, and this list should only show those counties with data waiting to be transcribed.  This list to be maintained by the CC.  The SC will get an email to say that they have a new transcriber, but it should also say the county in which the transcriber is interested.  The CC should get a copy so that they know that someone will be working on the images that they manage.  The SC can keep this email until training is complete, or they can use it to assign the appropriate images.

E0 U3 S7 C7: P17

**Comment by SteveBiggs** [comment]
I spend a fair amount of time moving userids from the county they signed up for into Training for Jo to look after them. Then when they've finished training, I move them into the county of their choice (that has data).

I think we should consider only allowing newbies to sign up directly into Training so they go straight to Jo. When they've finished training, they can be given the list of counties with data and pick one to be moved to.

**Comment by suffolkroots** [comment]
I agree Steve---newbies should sign up for training and go straight to Jo.

**Comment by richpomfret** [comment]
Agreed to go with Steve's suggestion there in terms of updating our process and going straight to training.

**Comment by stsccfr** [comment]
This is already implemented.
